common/models/acl.js

```diff
@@ -332,10 +337,11 @@ module.exports = function(ACL) {
     var accessTypeQuery = (accessType === ACL.ALL) ? undefined :
       {inq: [accessType, ACL.ALL, ACL.EXECUTE]};
 
-    var req = new AccessRequest(model, property, accessType);
+    var req = new AccessRequest({model, property, accessType, registry: this.registry});
 
     var acls = this.getStaticACLs(model, property);
 
+    // resolved is an instance of AccessRequest
     var resolved = this.resolvePermission(acls, req);
```

Comment:
further to commenting, i'd like to ask if ok (here or latter) to switch the name of the variable: resolved is just vague. This should be called something with accessRequest in it to make further code maintenance easier